packages/voila/index.js

```diff
@@ -0,0 +1,189 @@
+/***************************************************************************
```

Comment:
@jtpio can we convert this file into `typescript` and move it into the `src` directory?

Reply:
This file is inspired by the one in JupyterLab which corresponds to the entrypoint of the application. In the case of JupyterLab there is some templating with `handlebars` happening during the webpack build, which is probably why it can't be converted to TS there.

But it might be possible in the case of Voila yes.

Reply:
Thanks! I will try to do it.